libcloud/compute/drivers/gce.py

```diff
@@ -6816,7 +6831,7 @@ def ex_destroy_autoscaler(self, autoscaler):
         self.connection.async_request(request, method='DELETE')
         return True
 
-    def destroy_node(self, node, destroy_boot_disk=False):
+    def destroy_node(self, node, destroy_boot_disk=False, sync=True):
```

Comment:
I will go ahead and rename this to ``ex_sync`` since this is a "standard" API method so non-standard arguments should be prefixed with ``ex_`` (there is no ``ex_`` prefix needed with methods which already contain ``ex_`` prefix itself).